src/api/migraphx.py

```diff
@@ -248,6 +259,38 @@ def onnx_options(h):
                  returns='migraphx::program')
 
 
+@auto_handle()
+def tf_options(h):
+    h.constructor('create')
+    h.method(
+        'set_nhwc',
+        api.params(is_nhwc='unsigned char'),
```

Comment:
This should be `api.params(is_nhwc='bool')` as the parameter is a bool and not an unsigned char.

Reply:
fixed